.github/workflows/storybook.yml

```
@@ -14,8 +14,9 @@ on:
 
 jobs:
   build-storybook:
+    name: Build Storybook
     runs-on: ubuntu-latest
-    if: ${{ ! ( github.event_name == 'pull_request' && github.event.action == 'closed' ) }}
+    if: ${{ ! ( ( github.event_name == 'pull_request' && github.event.action == 'closed' ) || github.event.pull_request.head.repo.fork ) }}
```

Let's remove the expression syntax here, both for consistency (the job below does not use it) but also to make this a bit easier to read.
```suggestion
    if: ! ( ( github.event_name == 'pull_request' && github.event.action == 'closed' ) || github.event.pull_request.head.repo.fork )
```

---

@aaemnnosttv i don't remember why exactly I needed to add `${{ }}` but it had been done on purpose. Let's try to remove it and see if it can work without expression syntax.